.github/workflows/lint.yml

```diff
@@ -15,7 +11,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-node@v4
         with:
-          node-version-file: ".nvmrc"
+          node-version: "lts/Iron"
```

Comment:
Check this is the latest LTS version?

Can this be set to latest LTS?

Can this be updated with Dependabot or Renovate?

Reply:
#558